chain/client/tests/process_blocks.rs

```diff
@@ -825,3 +828,29 @@ fn test_minimum_gas_price() {
     let block = env.clients[0].chain.get_block_by_height(100).unwrap();
     assert!(block.header.inner_rest.gas_price >= min_gas_price);
 }
+
+#[test]
+fn test_garbage_collection() {
```

Comment:
1. Modify the test to accept `epoch_length` as a parameter.
2. At the end of the test please assert that all blocks in range `0..should_be_GCd` are deleted and all blocks in range `should_be_GCd..head` are exist.
3. Run the test with different `epoch_length` parameter.